pinot-core/src/test/java/org/apache/pinot/queries/DistinctQueriesTest.java

```diff
@@ -580,18 +578,16 @@ private void testDistinctInnerSegmentHelper(String[] queries, boolean isPql) {
    */
   @Test
   public void testDistinctInnerSegment() {
-    testDistinctInnerSegmentHelper(
-        new String[]{
-            "SELECT DISTINCT(intColumn, longColumn, floatColumn, doubleColumn, stringColumn, bytesColumn) FROM "
-                + "testTable LIMIT 10000",
-            "SELECT DISTINCT(stringColumn, bytesColumn, floatColumn) FROM testTable WHERE intColumn >= 60 LIMIT 10000",
-            "SELECT DISTINCT(intColumn, rawBytesColumn) FROM testTable ORDER BY rawBytesColumn LIMIT 5",
-            "SELECT DISTINCT(ADD(intColumn, floatColumn), stringColumn) FROM testTable WHERE longColumn < 60 ORDER BY"
-                + " stringColumn DESC, ADD(intColumn, floatColumn) ASC LIMIT 10",
-            "SELECT DISTINCT(floatColumn, longColumn) FROM testTable WHERE stringColumn = 'a' ORDER BY longColumn "
-                + "LIMIT 10"
-        },
-        true);
+    testDistinctInnerSegmentHelper(new String[]{
+        "SELECT DISTINCT(intColumn, longColumn, floatColumn, doubleColumn, stringColumn, bytesColumn) FROM "
+            + "testTable LIMIT 10000",
+        "SELECT DISTINCT(stringColumn, bytesColumn, floatColumn) FROM testTable WHERE " + "intColumn >= 60 LIMIT 10000",
+        "SELECT DISTINCT(intColumn, rawBytesColumn) FROM testTable ORDER BY " + "rawBytesColumn LIMIT 5",
```

Comment:
minor: Concatenation of two strings in these two lines can be replaced by a single string.

Reply:
done.